packages/channels/src/base/service.ts

```diff
@@ -112,9 +112,6 @@ export abstract class ChannelService<
     if (index) {
       const key = this.getIndexCacheKey(scope, identity, sender)
       const options = this.cacheIndexResponses.get(key)
-
-      this.cacheIndexResponses.del(key)
-
```

Comment:
Removing this since it makes sense that when you get a carousel in twilio, and there is a list of choices to make, you could make more than once choices successively. That line prevented you from doing that